examples/model_hosting/schedule_troubleshooting/ScheduleTroubleshooting.ipynb

```diff
@@ -619,8 +601,8 @@
     "})\n",
     "\n",
     "df = DataFetcher(ds).time_series.fetch_dataframe([\"inlet_pressure\", \"outlet_pressure\", \"cpr\"])\n",
-    "df.plot(x='timestamp', y=['inlet_pressure','outlet_pressure'])\n",
-    "df.plot(x='timestamp', y='cpr')"
+    "df.plot(y=['inlet_pressure','outlet_pressure'])\n",
+    "df.plot(y='cpr')"
```

Comment:
Don't want/need `x='timestamp'` anymore on those two lines?

Reply:
As above, not anymore.